### PR TITLE
Add "Throw & Go" support to Tello

### PR DESCRIPTION
**Pull request by cbwang2016**
Add Throw & Go support(Usage same as TakeOff() function)

### CODE REVIEW

File: platforms/dji/tello/driver.go [hunk 1]
```diff
@@ -279,6 +279,17 @@ func (d *Driver) TakeOff() (err error) {
 	return
 }
 
+// Throw & Go support 
+func (d *Driver) ThrowTakeOff() (err error) {
+	buf, _ := d.createPacket(0x5d, 0x48, 0)
```
Comment (deadprogram) [inline platforms/dji/tello/driver.go]: You should probably create a constant for the command byte here, to match the other commands.

Comment (cbwang2016) [reply platforms/dji/tello/driver.go]: Oh, sorry, so stupid of me. Can you help me to edit my code?

Comment (deadprogram) [reply platforms/dji/tello/driver.go]: If you make a change to your branch, commit it, and then push to Github, it should update this PR.

Comment (SMerrony) [reply platforms/dji/tello/driver.go]: Also, add the event and handle it...

Comment (deadprogram) [reply platforms/dji/tello/driver.go]: Good point @SMerrony yes, that is also needed.

Comment (cbwang2016) [reply platforms/dji/tello/driver.go]: @SMerrony There's no such event, I think. It's just FlightData.ThrowFlyTimer and receives with FlightDataEvent

**Comment by deadprogram** [comment]
Just out of curiously, how do you use this? You call it then throw the drone up in the air? :helicopter: 

I just want to make sure that is correct before I try it... perhaps a note in the GoDoc comment for the method would be good.

**Comment by cbwang2016** [comment]
Yep. 
1. Put your drone on your hands
2. Execute the ThrowTakeOff function, the propellers would rotates slowly
3. Throw your drone up in the air
You can try it on Windows using my app: https://github.com/cbwang2016/tello-control

**Comment by deadprogram** [comment]
Oops, when I merged the PR from @SMerrony it added a conflict with this one. @cbwang2016 can you resolve, or do you need assist?

**Comment by deadprogram** [comment]
Thank you. Now merging.

**Comment by cbwang2016** [comment]
@deadprogram When will you release a new version of Gobot?

**Comment by deadprogram** [comment]
@cbwang2016 we will likely have a Gobot release next week, to correspond with a parallel release of GoCV.

**Pull request merged**